components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -32,18 +33,24 @@ import {
 
 const TEAM_DETAILS_ROUTE = /^\/settings\/teams/;
 
+type TeamTabNames = 'users' | 'details';
```

Comment:
I violated a sort-of-standard notion that type names should be singular in my PR, and you've got the same here. Because when you instantiate something of this type, `const t: TeamTabNames` does not read right. That says "here's a constant t that is a TeamTabNames". Let's change this to singular.